packages/world/src/AccessControl.sol

```diff
@@ -7,11 +7,18 @@ import { IWorldErrors } from "./IWorldErrors.sol";
 import { ResourceAccess } from "./codegen/tables/ResourceAccess.sol";
 import { NamespaceOwner } from "./codegen/tables/NamespaceOwner.sol";
 
+/**
+ * @title AccessControl
+ * @dev Provides access control functions for checking permissions and ownership within a namespace.
+ */
 library AccessControl {
   using WorldResourceIdInstance for ResourceId;
 
   /**
-   * Returns true if the caller has access to the namespace or name, false otherwise.
+   * @notice Checks if the caller has access to the given resource ID or its namespace.
+   * @param resourceId The resource ID or namespace ID to check access for.
```

Comment:
```suggestion
   * @param resourceId The resource ID to check access for.
```

namespace ID is a type of resource ID (and we always check on both the namespace and individual resource anyway)